packages/oas-utils/package.json

```diff
@@ -101,7 +101,7 @@
     "@scalar/themes": "workspace:*",
     "@scalar/types": "workspace:*",
     "nanoid": "^5.0.7",
-    "yaml": "^2.4.5",
+    "yaml": "^2.5.1",
```

Comment:
Can we stick to the other version? I think it’d increase the bundle size otherwise because one dependency is pinned to the other version.

Reply:
I didn't look around enough to have an input on that, and it's not like this _needs_ to be 2.5.1, so 2.4.5 it is 👍